cvat-ui/src/components/annotation-page/standard-workspace/canvas-wrapper.tsx

```diff
@@ -161,6 +168,10 @@ export default class CanvasWrapperComponent extends React.PureComponent<Props> {
             });
         }
 
+        if (prevProps.showAllInterpolationTracks !== showAllInterpolationTracks) {
+            onFetchAnnotation();
+        }
+
```

Comment:
Can we dispatch this action immediately after changing settings?
Here: ``src/containers/header/settings-modal/workspace-settings.tsx`` Line 72

Reply:
@bsekachev, what about example future case when we change value by loading it from server? I think it can be more common way.

Reply:
@ActiveChooN 

I am not sure I understand you. What value? How are we going to get it from the server?
My idea is that now it is not obvious how the action is called after changing ``showAllInterpolationTracks``.